.github/workflows/main.yaml

```diff
@@ -29,8 +29,7 @@ jobs:
 
       - name: Build
         run: |
-          go build -mod=vendor -ldflags="-linkmode=external -extldflags=-static -s -w" cmd/agent/main.go
-          go build -mod=vendor -ldflags "-s -w" cmd/cli/main.go
+          go build -v ./...
```

Comment:
Is there a particular reason why you are changing linker flags?

Reply:
No, I replicated what was done on prism.